runhouse/main.py

```diff
@@ -836,12 +840,19 @@ def restart(
         help="Minimum log level for logs to be printed",
         callback=lambda value: value.upper(),
     ),
+    enable_telemetry_collection: bool = typer.Option(
```

Comment:
actually it's slightly more of an inconvenience to the user to do it this way, bc this is saying you need to provide the "--enable-telemetry-collection" flag explicitly in order for it to be enabled, but really it should be the opposite (enabled by default unless otherwise explicitly specified) 

so maybe we do: 
```
    parser.add_argument(
        "--disable-telemetry",
        action="store_true",  # if providing --disable-telemetry will turn off all telemetry / data collection
        default=argparse.SUPPRESS,  # If user didn't specify, attribute will not be present (not False)
        help="Whether to collect telemetry data",
    )
```